### DISCUSSION TITLE
11월 회의록

### DISCUSSION THREAD

**Discussion by KimJinHye0n** [gimdongwon/Catch_python_tmi#78] [Meeting Notes]
***2021.11.02***
---
[지각]
누적 : 김진현 (1스택)
[과제]
DFS : 트리까지

***2021.11.05***
---
[과제]
재귀 5문제 (~ 쿼드트리)

***2021.11.09***
---
[지각]
누적 : 김진현(1스택)
[과제]
재귀 3문제

***2021.11.16***
---
[지각]
X
[진행사항]
스택 문제 풀이 진행
[과제]
백트래킹 5문제

***2021.11.19***
---
[지각]
누적 : 나채빈(1스택)
[과제]
동적계획법 5문제

***2021.11.23***
---
[지각]
X
[과제]
DP 문제 더 풀기(5문제)
[시간]
오후 4시 30분 

***2021.11.25***
---
[지각]
누적 : 김진현(1스택)
[과제]
구현 5문제

**Comment by KimJinHye0n** [comment]
[1105] 이분그래프 / 텀프로젝트
[1109] 종이의개수 / Z / 색종이만들기 / 별찍기-11
[1116] 스택 / 제로 / 스택 수열 / 쇠막대기 / 에디터 / 탑 / 오큰수

**Comment by gimdongwon** [comment]
[1105] 촌수계산/이분그래프
[1109] 트리순회, 로또, 하노이

**Reply by gimdongwon** [reply]
11 street 대비 codility 문제 풀이중

**Reply by gimdongwon** [reply]
[1119] N과 M (1,2,3,4), 부분수열의합, 알파벳, 차이를 최대로

**Reply by gimdongwon** [reply]
[1123] [DP] 설탕배달, 1로만들기, 피보나치 함수, 1,2,3 더하기, 피보나치 수 5

**Comment by CHAEBEENA** [comment]
[1109] 로또 / 종이의개수
